app.py

```diff
@@ -0,0 +1,40 @@
+from flask import Flask, jsonify, request, Response, json
```

Comment:
`Response` and `json` is not used.
It's better to import only the necessary objects.